### PR TITLE
Documentation: Structure backup sections more clearly

### PR DESCRIPTION
**Pull request by quantenProjects**
## Proposed change

I got confused if the line about [database backup](https://github.com/paperless-ngx/paperless-ngx/blob/c19df84bef8d5678e2d49f4f802b67973d384a86/docs/administration.md?plain=1#L48) is relevant to all backup methods or only to the bare-metal folder copy method.
In the [online documentation](https://docs.paperless-ngx.com/administration/#backup) the indentation in the markdown source is not rendered.
Since the line in the md source is indented I assume it is only relevant to the bare-metal folder copy method. If this assumption is not correct, I would be happy to receive further information on how the interpret the sentence.

To emphasize the scope of the sentence I structured the documentation with additional headings.

Additionally, I removed the empty restore heading. For reference: this was the commit removing the content: https://github.com/paperless-ngx/paperless-ngx/commit/070f8ee949ae5b7bac8d56a0179b7c34fe032962#diff-8ea590b51774196952a802a56a908ec62639fcd8c8161566a48d700cf4a117cbL25-L65

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain): clarify documentation by more structure

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] ~~If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.~~
- [ ] ~~If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.~~
- [ ] ~~If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).~~
- [ ] ~~I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).~~
- [x] I have made corresponding changes to the documentation as needed.
- [ ] ~~I have checked my modifications for any breaking changes.~~

### CODE REVIEW

**Review by shamoon — CHANGES_REQUESTED** [review]
Thanks for PR. The thing is:

1. There are already a ton of headings in the sidebar (see attached), I kinda dont want to add more.
2. Perhaps a simpler solution would be to indent that one section about volumes, I agree that would make things clearer. But, to do so requires a change to our prettier settings, which, funny enough, is already in #4542 for the same reason
3. Agree with removing the Restoring header, or perhaps linking to the importer

Screenshot 2023-11-11 at 2 26 23 PM

**Comment by quantenProjects** [comment]
@shamoon thanks for the comments

Indenting the relevant sentence would appropriate in my opinion :+1: I would suggest to wait how #4542 turns out and then close this if it works out.

I would add the link to the importer in the section about the exporter backup, since the importer is only applicable to the exporter backup.

**Comment by shamoon** [comment]
@quantenProjects that other PR is merged (well technically will be soon) if you’d like to rebase this and make the changes we discussed. Thanks so much

**Comment by shamoon** [comment]
I went ahead and made the changes discussed, let us know if you have any concerns

**Comment by quantenProjects** [comment]
Sorry I haven't had time to do it in the last few days. 

Thank you very much for the changes. They look good to me :+1: 

Just made a minor change, since the incremental backup is just a specification of the exporter backup and not a different type. From my side it's good to merge.

One more minor thing: I wondered if the "Before making backups, make sure that paperless is not running." sentence is applicable to the exporter backup or only to the volume and file copy methods, but don't have enough inside into the inner workings of the exporter tool.

File: docs/administration.md [hunk 1]
```diff
@@ -25,31 +26,37 @@ Options available to any installation of paperless:
...
 
-- Backup the docker volumes. These usually reside within
-  `/var/lib/docker/volumes` on the host and you need to be root in
-  order to access them.
+-   Backup the docker volumes. These usually reside within
+    `/var/lib/docker/volumes` on the host and you need to be root in
+    order to access them.
 
-  Paperless uses 4 volumes:
+    Paperless uses 4 volumes:
```
Comment (dblitt) [inline docs/administration.md:33]: We should probably also note that as of #132 paperless also uses a `paperless_redisdata` volume for the broker (although to be fair, I don't think much important data is stored there)

Comment (shamoon) [reply docs/administration.md:33]: I dont think theres any reason to back that up, its all ephemeral data that gets recreated

**Comment by shamoon** [comment]
> One more minor thing: I wondered if the "Before making backups, make sure that paperless is not running." sentence is applicable to the exporter backup or only to the volume and file copy methods, but don't have enough inside into the inner workings of the exporter tool.

Actually might as well take the opportunity update this, its been a point of confusion, I'll change it to reflect what it means which is more like "make sure youre not actively consuming documents"

**Review by shamoon — APPROVED** [review]
Great, thanks again

**Pull request merged**